src/components/displaySettings/displaySettings.template.html

```diff
@@ -224,7 +224,7 @@ <h2 class="sectionTitle">
     </h2>
 
     <div class="inputContainer inputContainer-withDescription">
-        <input is="emby-input" type="number" id="txtLibraryPageSize" pattern="[0-9]*" required="required" min="0" max="1000" step="1" label="${LabelLibraryPageSize}" />
+        <input is="emby-input" type="number" id="txtLibraryPageSize" pattern="[0-9]*" required="required" min="0" max="10000" step="1" label="${LabelLibraryPageSize}" />
```

Comment:
10k is really not a practical limit for this. It will cause pages to load too slow especially on lower end servers.

Reply:
Hmm, ya, I get that. I also get letting the user define a setting that could overload the server isn't good. I didn't really think about that. 

So 1000 is too small if you use the many free legal IPTV sources since it shows times for shows multiple times. Maybe a grouping system with sub lists might be a better option and grouping by show will make these lists more user friendly as the end goal is to find content you want to watch in the future doesn't really matter when it is playing. I will think on this and make a PR once I get a good solution. That is out of scope for this PR though so I am good with leaving the user defined max as it was.

Reply:
Is this just a workaround for pagination not working correctly then?

Reply:
> Is this just a workaround for pagination not working correctly then?

No specific change was an enhancement on top of fixing the pagination. The other change in lists.js should be a fix for the issue I describe in my ticket. 

Actually I just realized that I missed adding the fix for user config librarypagesize = 0. Committing that now.